presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java

```diff
@@ -66,21 +68,21 @@ public static DistributedQueryRunner createQueryRunner(TpchTable<?>... tables)
     public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables)
             throws Exception
     {
-        return createQueryRunner(tables, ImmutableMap.of());
+        return createQueryRunner(tables, ImmutableMap.of(), Optional.empty());
     }
 
-    public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables, Map<String, String> extraProperties)
+    public static DistributedQueryRunner createQueryRunner(Iterable<TpchTable<?>> tables, Map<String, String> extraProperties, Optional<Path> baseDirPath)
             throws Exception
     {
-        return createQueryRunner(tables, extraProperties, "sql-standard", ImmutableMap.of());
+        return createQueryRunner(tables, extraProperties, "sql-standard", ImmutableMap.of(), baseDirPath);
```

Comment:
- In other parts of the code we use `baseDataDir`, so we should be consistent.
- In general we don't use abbreviations in Presto, so I think we can do the same here and rename these variables to `baseDataDirectory`.

Reply:
Also, what does `base` mean here? Why don't we rename this variable to `dataDirectory` or `localDataDirectory` if the data is only supposed to touch the local filesystem?

Reply:
@nezihyigitbasi It's called baseXXX because there will be a hive_data folder created under it. In TestingPrestoServer, it used to be named as baseDataDir and getBaseDataDir():
    ```
    public Path getBaseDataDir()
    {
        return baseDataDir;
    }
```
So I suggest we still call it baseDataDir.  If we want to rename it, we can do it in a separate commit.